src/modules/rewards/components/RewardsActivitiesList.tsx

```diff
@@ -20,7 +20,7 @@ const RewardsActivitiesList: FC<RewardActivitiesProps> = () => {
 
   const { data: rewardActivitiesResponse, isLoading: isLoadingActivities } = useGetRewardsActivities();
 
-  const filteredActivities = (rewardActivitiesResponse?.activities || []).filter((activity) => activity.activityType !== 'follow_push_on_twitter');
+  const filteredActivities = (rewardActivitiesResponse?.activities || []);
```

Comment:
Why are we still using the name filteredActivities if we are not filtering out anything?

Reply:
```suggestion
  const activityList = isLoading ? Array(3).fill(0) : rewardActivitiesResponse?.activities;
```